lightkurve/tests/test_search.py

```diff
@@ -85,6 +85,31 @@ def test_search_lightcurvefile(caplog):
     assert(len(search_lightcurvefile("pi Mensae", sector=1).table) == 1)
 
 
+@pytest.mark.remote_data
+def test_search_cutout():
+    # Cutout by target name
+    assert(len(search_cutout("pi Mensae", sector=1).table) == 1)
+    # Cutout by TIC ID
+    assert(len(search_cutout('TIC 206669860', sector=2).table) == 1)
+    # Cutout by RA, dec string
+    search_string = search_cutout('30.578761, -83.210593')
+    # Cutout by SkyCoord
+    c = SkyCoord('30.578761 -83.210593', unit=(u.deg, u.deg))
+    search_coords = search_cutout(c)
+    # These should be identical
+    assert(len(search_string.table) == len(search_coords.table))
+    # Make sure they can be downloaded with default size
+    tpf = search_string.download()
+    # Ensure the correct object has been read in
+    assert(isinstance(tpf, TessTargetPixelFile))
+    # Ensure default size is 5x5
+    assert(tpf.flux[0].shape == (5,5))
+    # Download with different dimensions
+    tpf = search_coords.download(cutout_size=4)
+    # Ensure correct dimensions
+    assert(tpf.flux[0].shape == (4,4))
+
```

Comment:
Only other test I can suggest is trying to download the "edge" of an FFI and checking that an exception is raised.